### ISSUE TITLE
Assets with orange color in color variants.

### ISSUE THREAD

**Issue by Initu-Castilhos** [ubuntu/yaru#3555]
Assets with orange color in color variants as shown in the image below.
![Captura de tela de 2022-03-24 12-59-57](https://user-images.githubusercontent.com/59381215/159961056-d301fb44-9a75-44dd-9a9a-5265cdae16e0.png)

My suggestion is to use monochromatic color (gray).

**Comment by 3v1n0** [comment]
What app is that? What toolkit?

**Comment by Muqtxdir** [comment]
this happens whenever `GtkScale`'s slider is dragged (_noticed in sound panel, just now_)

**Comment by Initu-Castilhos** [comment]
> Que aplicativo é esse? Que kit de ferramentas?

dock size adjustment.

**Comment by Jupi007** [comment]
I'm not sure, but I think this looks like the Gtk4 slider.

**Comment by Initu-Castilhos** [comment]
> Não tenho certeza, mas acho que isso se parece com o controle deslizante de escala Gtk4.

![Captura de tela de 2022-03-26 21-27-41](https://user-images.githubusercontent.com/59381215/160261517-91c25748-5d64-40a2-a31b-e67182a5d268.png)

**Comment by Jupi007** [comment]
I was wrong, this is indeed the Gtk3 version.

**Comment by 3v1n0** [comment]
Yeah, the asset svgs needs to be re-generated too... Will handle this ASAP:
